src/Assignment-3/assignment3.css

```diff
@@ -1,7 +1,58 @@
 /* Our CSS file. */
 
+.container,
+.content {
+  width: 100%;
+  height: 100%;
+}
+.container1 {
+  display: flex;
+  background-color: #f2f2f2;
+}
+.para1 {
+  width: 50%;
+  padding: 80px;
+  display: inline-block;
+}
+.para2 {
+  padding-left: 250px;
+  padding-right: 250px;
+  display: block;
+}
+
+.frame1 {
+  width: 50%;
+  display: inline-block;
+  padding: 30px;
+}
+.container2 {
+  background-color: #f2f2f2;
+}
+.frame2 {
```

Comment:
iframe width shold be 100% of the parent container. Hardcoded values are not responsive.